src/Catel.MVVM/Catel.MVVM.Shared/Collections/FastObservableCollection.cs

```diff
@@ -31,6 +34,31 @@ public class FastObservableCollection<T> : ObservableCollection<T>, ISuspendChan
 
         #region Fields
         private bool _suspendChangeNotifications;
+
+        /// <summary>
+        /// Added items while suspending notifications.
+        /// </summary>
+        private readonly List<T> _newItems = new List<T>();
```

Comment:
Maybe it's better to create a SuspensionContext that contains this items. The deltas actually belong to a suspension, not to the actual collection itself?